chore: Update version constraints and add MIGRATING docs (#483)
DataDog/dd-sdk-flutter: CONTRIBUTING.md

SDK](https://docs.flutter.dev/get-started/install), and that `flutter doctor`
passes without issues.

> The Datadog packages uses [Melos](https://github.com/invertase/melos) to manage

Reviewer: is this a right change?

Author: Nope. Good call.

melos bootstrap
```

> Next, we need to generate some necessary files with `flutter pub run build_runner build`, a
> nd generate `.env` files for the various apps in order to use them with Datadog. We use Melos 

Reviewer: ```suggestion
Next, we need to generate some necessary files with `flutter pub run build_runner build`, and
 generate `.env` files for the various apps in order to use them with Datadog. We use Melos 
```
